### ISSUE TITLE
v0.99 - Modal - options dismissible doesn't work without link

### ISSUE THREAD

**Issue by storm1er** [Dogfalo/materialize#4909]
Modal created dynamically ~~without link~~ are dismissible even with `dismissible = false`

[See CodePen](https://codepen.io/anon/pen/QgaoYG)
```javascript
$( document ).ready(function() {
	$('body').append(
		'<div id="modalproblem" class="modal">'+
			'<div class="modal-content">'+
				'<h4>Test modal</h4>'+
				'<p class="flow-text" style="line-height:1em;">'+
					'<span style="font-size:.8em;">'+
						'This modal should not be dismissible'+
					'</span>'+
				'</p>'+
			'</div>'+
		'</div>'
	);
	$('#modalproblem').modal({
		dismissible:false
	});
	$('#modalproblem').modal('open');
});
```

**Comment by storm1er** [comment]
well, link or not it seems to be the same : dismissible doesn't work

**Comment by storm1er** [comment]
Corrected with this https://github.com/storm1er/materialize/commit/3f72c750438dac16e6544df4803af287bd38ea60

**Comment by acburst** [comment]
I've made a compromise to keep the old functionality but fix this bug

Fixed in 8555d149
